src/languages/de.ts

```diff
@@ -2552,13 +2552,13 @@ ${amount} für ${merchant} - ${date}`,
                         4. Suche nach ${integrationName}.
                         5. Klicke auf *Connect*.
 
-${
-    integrationName && CONST.connectionsVideoPaths[integrationName]
-        ? dedent(`[Zur Buchhaltung](${workspaceAccountingLink}).
+                        ${
+                            integrationName && CONST.connectionsVideoPaths[integrationName]
+                                ? `[Zur Buchhaltung](${workspaceAccountingLink}).
```

Comment:
@bernhardoj as we are taking out `dedent` from here, don't we need to remove it from en.ts too?

https://github.com/Expensify/App/blob/1cc18e7eea83e5a50294b65a3c75cb158c4cc1b9/src/languages/en.ts#L2653